pages/setuppwa/setuppwa.tsx

```diff
@@ -0,0 +1,73 @@
+import { useRouter } from "next/router"
+import React from "react"
+
+const SetupPwa = () => {
+  const router = useRouter()
+  const [username, setUsername] = React.useState<string>("")
+
+  let accountUsername: string
+  const username_from_local = localStorage.getItem("username")
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault()
+
+    if (accountUsername === "" || !accountUsername) {
```

Comment:
What is the goal with this conditional `accountUsername === "" || !accountUsername`? Currently it will write an empty string or a falsy value to localstorage.